bindata/network/additional-networks/crd/001-crd.yaml

```diff
@@ -13,3 +13,68 @@ spec:
     kind: NetworkAttachmentDefinition
     shortNames:
     - net-attach-def
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: ippools.whereabouts.cni.k8s.io
+spec:
+  group: whereabouts.cni.k8s.io
+  names:
+    kind: IPPool
+    plural: ippools
+  scope: ""
+  validation:
+    openAPIV3Schema:
+      description: IPPool is the Schema for the ippools API
```

Comment:
The documentation here is weird; it's there, but it's not enough to have any idea what any of this means unless you already know...

Reply:
Agreed, it's like using the word in a definition, I'll touch this up.